examples/resources/biganimal_tag/resource.tf

```diff
@@ -0,0 +1,20 @@
+terraform {
+  required_providers {
+    biganimal = {
+      source  = "EnterpriseDB/biganimal"
+      version = "0.8.1"
+    }
+    random = {
+      source  = "hashicorp/random"
+      version = "3.6.0"
+    }
+  }
+}
+
+resource "biganimal_tag" "tag" {
+  tag_name = "tag-name"
+  // The default colors to choose from are:
+  // magenta, red, orange, yellow, green, teal, blue, indigo, purple and grey
```

Comment:
have we defined map for color name and color hex code or does API is able to handle  color names?